diff --git a/common/car.mk b/common/car.mk
index 1a98679..b9eefdc 100644
--- a/common/car.mk
+++ b/common/car.mk
@@ -17,7 +17,6 @@
 # Auto modules
 # TODO: Add broadcastradio@.2.0 back once it's stable b/145694104
 PRODUCT_PACKAGES += \
-    android.hardware.automotive.vehicle@2.0-service \
     android.hardware.automotive.audiocontrol@2.0-service \
     android.frameworks.automotive.display@1.0-service \
 
